src/functionalTest/java/uk/gov/hmcts/reform/pip/account/management/controllers/InactiveAccountsManagementTest.java

```diff
@@ -0,0 +1,277 @@
+    private static final String TEST_EMPLOYER = "E2E Account Management Test Employer";
+    private static final Integer TEST_EMAIL_RANDOM_NUMBER =
+        ThreadLocalRandom.current().nextInt(1000, 9999);
+    private static final String TEST_EMAIL_PREFIX = String.format(
+        "pip-am-test-email-%s", TEST_EMAIL_RANDOM_NUMBER);
+    private static final String TEST_ADMIN_EMAIL_PREFIX = String.format(
+        "pip-am-test-admin-email-%s", TEST_EMAIL_RANDOM_NUMBER);
+    private static final String TEST_IDAM_EMAIL_PREFIX = String.format(
+        "pip-am-test-idam-email-%s", TEST_EMAIL_RANDOM_NUMBER);
+    private static final String EMAIL_DOMAIN = "%s@justice.gov.uk";
+    private static final String TEST_EMAIL =
+        String.format(EMAIL_DOMAIN, TEST_EMAIL_PREFIX);
+    private static final String TEST_ADMIN_EMAIL =
+        String.format(EMAIL_DOMAIN, TEST_ADMIN_EMAIL_PREFIX);
+    private static final String TEST_IDAM_EMAIL =
+        String.format(EMAIL_DOMAIN, TEST_IDAM_EMAIL_PREFIX);
+    private static final String FIRST_NAME = "E2E Account Management";
+    private static final String SURNAME = "Test Name";
+    private static final String STATUS = "PENDING";
+    private static final String LAST_SINGED_IN_DATE = "lastSignedInDate";
+    private static final String MEDIA_APPLICATION_URL = "/application";
+    private static final String MOCK_FILE = "files/test-image.png";
+    private static final String BEARER = "Bearer ";
+    private static final String ISSUER_ID = "x-issuer-id";
+    private static final Clock CL = Clock.systemUTC();
+    private static final String IDAM_USER_PROVENANCE_ID = UUID.randomUUID().toString();
+    private Map<String, String> headers;
+    private Map<String, String> issuerId;
+    private String mediaUserProvenanceId;
+    private String adminProvenanceId;
+    private static final String ACCOUNT_URL = "/account";
+    private static final String ADD_USER_B2C_URL = ACCOUNT_URL + "/add/azure";
+    private static final String ADD_PI_USER_URL = ACCOUNT_URL + "/add/pi";
+    private static final String UPDATE_USER_INFO = ACCOUNT_URL + "/provenance";
+    private static final String NOTIFY_INACTIVE_MEDIA_ACCOUNT = ACCOUNT_URL + "/media/inactive/notify";
+    private static final String DELETE_INACTIVE_MEDIA_ACCOUNT = ACCOUNT_URL + "/media/inactive";
+    private static final String NOTIFY_INACTIVE_ADMIN_ACCOUNT = ACCOUNT_URL + "/admin/inactive/notify";
+    private static final String DELETE_INACTIVE_ADMIN_ACCOUNT = ACCOUNT_URL + "/admin/inactive";
+    private static final String NOTIFY_INACTIVE_IDAM_ACCOUNT = ACCOUNT_URL + "/idam/inactive/notify";
+    private static final String DELETE_INACTIVE_IDAM_ACCOUNT = ACCOUNT_URL + "/idam/inactive";
+    private static final String ADD_SYSTEM_ADMIN_B2C_URL = ACCOUNT_URL + "/add/system-admin";
+
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    @BeforeAll
```

Comment:
We also need an AfterAll to clean up / delete the accounts that have been created